src/container/srv_epoch.c

```diff
@@ -506,3 +469,61 @@ ds_cont_get_snapshots(uuid_t pool_uuid, uuid_t cont_uuid,
 
 	return rc;
 }
+
+/*
+ * Propagate new snapshot list to all servers through snapshot IV, errors
+ * are ignored.
+ */
+void
+ds_cont_update_snap_iv(struct ds_pool *pool, uuid_t cont_uuid)
```

Comment:
Could the pool parameter be replaced with a svc parameter? The IV NS is available with svc->cs_pool->sp_iv_ns. The pool UUID is available with svc->cs_pool_uuid.

Reply:
Sure, thanks.